release/serve_tests/workloads/single_deployment_1k_noop_replica.py

```diff
@@ -92,7 +92,7 @@ def main(
     # Give default cluster parameter values based on smoke_test config
     # if user provided values explicitly, use them instead.
     # IS_SMOKE_TEST is set by args of releaser's e2e.py
-    smoke_test = os.environ.get("IS_SMOKE_TEST", "1")
+    smoke_test = os.environ.get("IS_SMOKE_TEST", "0")
```

Comment:
@simon-mo @krfricke I think it's also safer if test infra always set IS_SMOKE_TEST=0 if it's not a smoke test.